blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js

```diff
@@ -2,5 +2,6 @@ import EditFormNewRoute from 'ember-flexberry/routes/edit-form-new';
 
 export default EditFormNewRoute.extend({
   modelProjection: '<%=modelProjection%>',
-  modelName: '<%=modelName%>'
+  modelName: '<%=modelName%>',
+  templateName: '<%=entityName%>',
```

Comment:
It [`'formName'`](https://github.com/Flexberry/ember-flexberry/blob/develop/vendor/flexberry/dummy-metadata/edit-forms/ember-flexberry-dummy-application-user-edit.json#L4), `'entityName'` set when command is invoked, example:
`ember generate flexberry-edit-form ember-flexberry-dummy-application-user-edit --metadata-dir=vendor/flexberry/dummy-metadata`